SpeechService/Humanize.cs

```diff
@@ -1,15 +1,24 @@
 ﻿using System;
+using System.Globalization;
 
 namespace EddiSpeechService
 {
     public partial class Translations
     {
-        public static string Humanize(decimal? rawValue)
+        /// <summary>
+        /// Present a number's approximate value in a format suitable for text-to-speech
+        /// </summary>
+        /// <param name="rawValue">The value to be formatted. If null, returns null.</param>
+        /// <param name="forceIntegerMantissa">true: always express the mantissa as an integer; false: allow the mantissa to be a short decimal; null: lookup behavior from localized resource</param>
+        /// <returns>A string being the number's approximate value in a format suitable for text-to-speech</returns>
+        public static string Humanize(decimal? rawValue, bool? forceIntegerMantissa = null)
         {
             if (rawValue == null) {return null;}
-            decimal value = (decimal) rawValue;
+            decimal value = (decimal)rawValue;
             if (value == 0) {return Properties.Phrases.zero;}
 
+            bool wantIntegerMantissa = forceIntegerMantissa ?? Properties.FormatOverrides.forceIntegerMantissa.Equals("true");
```

Comment:
Consider adding the secondary optional argument `StringComparison.InvariantCultureIgnoreCase` to `Equals(string value, StringComparison comparisonType)` to make this user entered parameter a little more forgiving?

Reply:
Ideally we'd use a single bit here for our flag, but RESX only caters for strings. I considered 1 and 0 here but eventually went with true and false for human-readability.

That said, I am not sure why you want to make it more forgiving given that it's a dev flag? My instinct would be in fact to make it stricter with `StringComparison.Ordinal`. This link is helpful https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings

Reply:
The intent is simply to make sure that a casing issue doesn't work against the translator. `StringComparison.OrdinalIgnoreCase` would work. Or "1" and "0".